### PR TITLE
Notification bell Added

### PR DESCRIPTION
**Pull request by TanmayKalra09**
Issue: https://github.com/AlfiyaSiddique/TastyTrails/issues/20
I have added the notification bell beside the login/signup, and it redirects the page to http://localhost:5173/notifications
image

### CODE REVIEW

**Comment by AlfiyaSiddique** [comment]
Hey @TanmayKalra09 
What about building the notification page aren't you going to do it?
Also notification bell should only appear if the user is logged in, Please fix it.

**Comment by TanmayKalra09** [comment]
> Hey @TanmayKalra09 What about building the notification page aren't you going to do it? Also notification bell should only appear if the user is logged in, Please fix it.

Hey @AlfiyaSiddique I have done the change that notification bell would only display if the user is logged in, can you please specify more about the notification page?

**Comment by AlfiyaSiddique** [comment]
The notification page should have all the notifications related to that user for example if someone likes the user's recipe or someone dislikes a recipe or if someone follows. These all notifications will be shown on the Notification page Just like Instagram.
@TanmayKalra09  Let me know if you can do this or not.

**Comment by TanmayKalra09** [comment]
Sorry, @AlfiyaSiddique this would be a little difficult for me :(
I need some time for this issue, I have to do some research about this. At this point, I won't be able to do such modifications.

**Comment by AlfiyaSiddique** [comment]
Okay no worries @TanmayKalra09 , will be merging this PR by EOD with only the notification icon and page addition.
Thanks for your contribution:)

File: frontend/src/Components/Navbar.jsx [hunk 1]
```diff
@@ -88,6 +90,9 @@ const Navbar = () => {
                 <FontAwesomeIcon icon={faPenSquare} className="mx-2 text-red-700 text-lg" />
                 <span className="text-black font-semibold">Add Recipe</span>
               </div>
+              <div onClick={() => navigator(`/notifications`)} className="ml-4 cursor-pointer">
```
[the remark this answers is not in the document]
Comment (AlfiyaSiddique) [reply frontend/src/Components/Navbar.jsx:93]: @TanmayKalra09 
Merging this PR now
But please take care from next time to describe well in issue till where you are implementing the feature
Thanks for your contribution

**Review by AlfiyaSiddique — APPROVED** [review]

**Pull request merged**